matrix-meetings-widget/src/setupTests.ts

```diff
@@ -86,3 +86,9 @@ global.TextDecoder = TextDecoder as typeof global.TextDecoder;
 Object.defineProperty(navigator, 'clipboard', {
   value: { writeText: jest.fn() },
 });
+
+// Polyfill structuredClone
+// @todo may become unnecessary after Vite migration
```

Comment:
A little odd we need this since its supported basically since ~ 10 years or so in everything. Do we have our typescript or webpack misconfigured? 🫠

Reply:
jsdom does not provide `structuredClone` ([reference](https://github.com/jsdom/jsdom/issues/3363))

Reply:
ah. I forgot jsdom is a thing. Thats surprisingly sad. Cant we polyfill it to the nodejs thing though?